app/gist/controller.js

```diff
@@ -113,6 +113,7 @@ export default Ember.Controller.extend({
         controller: this
       }));
     }
+    columns.objectAt(0).set('active', true);
```

Comment:
Column#active is a computed property with no setter.

The correct way to make a column active is to set activeEditorCol on the controller.